ext/oauth-client/Civi/Api4/OAuthContactToken.php

```diff
@@ -15,12 +15,7 @@ class OAuthContactToken extends Generic\DAOEntity {
   public static function permissions(): array {
     return [
       'meta' => ['access CiviCRM'],
-      'default' => [
-        [
-          'manage my OAuth contact tokens',
-          'manage all OAuth contact tokens',
-        ],
-      ],
+      'default' => ['manage my OAuth contact tokens'],
```

Comment:
I think the new "implies" relation makes sense.

This part is a sort of DRY refactoring. Looks safe.

Subjectively... if I were just skimming `OAuthContactToken.php`, I would understand the usage+behavior more quickly with the longer declaration. It tells me that there is a normal/expected mode of interaction based on `manage all OAuth contact tokens`. With the shorter declaration, I might (mistakenly) conclude that the API's *only intended usage* is to manage a personal token.

But again, it looks functionally safe. If you're still keen, then it's mergeable.